Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN apt-get install \
     python3-apt -y
 
 # Download Bazel
-RUN go get github.com/bazelbuild/bazelisk
+RUN go install github.com/bazelbuild/bazelisk@latest
```

Comment:
Why was this changed needed?

Reply:
`go get` is no longer supported, the compiler recommended using `go install`. It was also failing without the use of `@latest`